spec/support/features/clearance_helpers.rb

```diff
@@ -0,0 +1,25 @@
+module Features
+  module ClearanceHelpers
+    def sign_in
+      user = create(:user)
+      sign_in_with user.email, user.password
+    end
+
+    def sign_in_with(email, password)
+      visit sign_in_path
+      fill_in "session_email", with: email
+      fill_in "session_password", with: password
+      click_button I18n.t("helpers.submit.session.submit")
```

Comment:
I don't know if this comes included in Suspenders by default, but you should be able to call translation helpers by saying `t` instead of `i18n.t` in tests. If that doesn't work, then you can put

``` ruby
config.include AbstractController::Translation
```

inside of the `RSpec.configure` block in rails_helper.rb.